indexer/services/roundtable/src/helpers/adjectives.json

```diff
@@ -149,10 +149,8 @@
   "hip",
   "holy",
   "honest",
-  "hot",
```

Comment:
Refactored both word lists and spot checked 34 [proposed whitelist addresses ](https://www.mintscan.io/dydx/proposals/183)